libs/vkd3d/platform.c

```diff
@@ -146,6 +181,22 @@ const char *vkd3d_dlerror(void)
     return "Not implemented for this platform.";
 }
 
+bool vkd3d_get_env_var(const char *name, char *value, size_t value_size)
```

Comment:
Is this a duplicate of line 46?

Reply:
If this is intended to be a fallback for non-Linux/non-Win32, the getenv() path should be in a common `static inline bool vkd3d_get_env_var_stdlib()` or something like that and called from non-win32 paths. Alternatively, move out the code out of the platform specific ifdefs to avoid duplicates:

```
#ifdef _WIN32
bool vkd3d_get_env_var() { GetEnvironmentA(); }
#else
bool vkd3d_get_env_var() { getenv(); }
#endif
```

Reply:
Done